scripts/us_census/pep/monthly_population_estimate/preprocess.py

```diff
@@ -378,30 +401,260 @@ def process(self):
+        "Resident Population Plus Armed Forces Overseas", "Civilian Population",
+        "Civilian NonInstitutionalized Population"
+    ]
+    df.columns = cols
+    for col in df.columns:
+        df[col] = df[col].str.replace(",", "")
+    return df
 
-    loader = CensusUSACountryPopulation(ip_files, cleaned_csv_path, mcf_path,
-                                        tmcf_path)
 
-    loader.process()
+def _clean_txt_file(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    This method cleans the dataframe loaded from a txt file format.
+    Also, Performs transformations on the data.
+
+    Arguments:
+        df (DataFrame): DataFrame representing the loaded TXT dataset.
+    
+    Returns:
+        DataFrame: Transformed DataFrame after cleaning operations.
+    """
+    df['Year and Month'] = df[['Year and Month', 'Date']]\
+                                    .apply(_concat_cols, axis=1)
+    df.drop(columns=['Date'], inplace=True)
+    for col in df.columns:
+        df[col] = df[col].str.replace(",", "")
+
+    # The index numbers alotted as per where the columns are present to
+    # move the columns left
+    resident_population = 1
+    resident_population_plus_armed_forces_overseas = 2
+    civilian_population = 3
+    civilian_noninstitutionalized_population = 4
+    # Moving the row data left upto one index value.
+    # As the text file has (census) mentioned in some rows and it makes the
+    # other column's data shift by one place, we need to shift it back to the
+    # original place.
+    idx = df[df['Resident Population'] == "(census)"].index
+    df.iloc[idx, resident_population] = df.iloc[idx][
+        "Resident Population Plus Armed Forces Overseas"]
+    df.iloc[idx, resident_population_plus_armed_forces_overseas] = df.iloc[idx][
+        "Civilian Population"]
+    df.iloc[idx, civilian_population] = df.iloc[idx][
+        "Civilian NonInstitutionalized Population"]
+    df.iloc[idx, civilian_noninstitutionalized_population] = np.NAN
+    return df
+
+
+def _mulitply_scaling_factor(col: pd.Series) -> pd.Series:
+    """
+    This method multiply dataframe column with scaling factor.
+
+    Arguments:
+        col (Series): A DataFrame column of dtype int, containing the values to be scaled.
+
+    Returns:
+        Series: A DataFrame column with values multiplied by the scaling factor.
+    """
+    res = col
+    if col not in [None, np.NAN]:
+        if col.isdigit():
+            res = int(col) * _SCALING_FACTOR_TXT_FILE
+    return res
+
+
+def _concat_cols(col: pd.Series) -> pd.Series:
+    """
+    This method concats two DataFrame column values
+    with space in-between.
+
+    Args:
+        col (Series): A pandas Series containing two values from the DataFrame.
+
+    Returns:
+        res (Series) : Concatenated DataFrame Columns
+    """
+    res = col[0]
+    if col[1] is None:
+        return res
+    res = col[0] + ' ' + col[1]
+    return res
+
+
+def download_files():

```

Comment:
Please add function block comments for this method.

Reply:
added